### ISSUE TITLE
Update tools to use new API format & environments

### ISSUE THREAD

**Issue by grahamboree** [giantotter/giantotter_public#403]
All the Unity-side calls aren't using the old-style API, and the teamtools are mostly moved over.  Is there anything else that's using these?  The flash tools?  Maybe some old devtools or libOtter stuff?  We should update or remove the scripts that are referencing these old api calls to work with the new format and multiple environment setup.

**Comment by jefforkin** [comment]
Maybe as a first step, we should make the old API return a recognizable error, so that we can fix things as we encounter them.

The flash tools likely are using the old api, but there is another issue (243) to revisit those anyway.  But not until we've tackled 244 importing event tags.

**Comment by grahamboree** [comment]
Quick update: the only things left using the old API style are the flash tools.  The rest has moved over to the new format.

**Comment by grahamboree** [comment]
Turns out there are some ancient teamtools stuff that also uses the old api format.

**Comment by jefforkin** [comment]
Is there anything left to do on this?

**Comment by grahamboree** [comment]
Dunno.  I'll add it to my list and take a look

**Comment by jefforkin** [comment]
no longer relevant
